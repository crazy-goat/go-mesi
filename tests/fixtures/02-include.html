<!DOCTYPE html>
<html>
<head>
    <title>Standard esi:include tests</title>
</head>
<body>
This should print hello world: [<esi:include src="http://127.0.0.1:8080/hello" />]
This should print 404: [<esi:include src="http://127.0.0.1:8080/status/code/404" />]
This should print 500: [<esi:include src="http://127.0.0.1:8080/status/code/404" alt="http://127.0.0.1:8080/status/code/500"/>]
This should be empty: [<esi:include onerror="continue" src="http://127.0.0.1:8080/status/code/500" />]
This should also be empty: [<esi:include onerror="continue" src="http://127.0.0.1:8080/status/code/500" alt="http://127.0.0.1:8080/status/code/500"/>]
This should print hello world: [<esi:include onerror="continue" src="http://127.0.0.1:8080/status/code/500" alt="http://127.0.0.1:8080/hello"/>]
</body>
</html>
