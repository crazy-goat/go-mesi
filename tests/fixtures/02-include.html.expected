<!DOCTYPE html>
<html>
<head>
    <title>Standard esi:include tests</title>
</head>
<body>
This should print hello world: [Hello World]
This should print 404: [404: Not Found]
This should print 500: [500: Internal Server Error]
This should be empty: []
This should also be empty: []
This should print hello world: [Hello World]
</body>
</html>
